cpp/inc/bond/core/transforms.h

```diff
@@ -426,17 +432,31 @@ class RequiredFieldValiadator
 private:
     BOND_NORETURN void MissingFieldException() const;
 
+    BOND_NORETURN void UnexpectedStructStopException() const;
+
     mutable uint16_t _required;
 };
 
 template <typename T>
 void RequiredFieldValiadator<T>::MissingFieldException() const
 {
+    (void)typename schema<T>::type();
```

Comment:
Why do we need to construct an instance of the field?

Reply:
To force-instantiate a possibly template static data member `metadata` (see [this link error](https://ci.appveyor.com/project/MicrosoftBond/bond/build/1.0.1048/job/8o6t55ekn6lqoq0s)).

Reply:
Can we comment the line?